docs/src/content/faq.mdx

```diff
@@ -48,27 +81,275 @@ After receiving the NFT, follow these steps to set up the Icebreaker platform:
+
+Regardless of your approach, refer to the [Getting started](get-started) documentation to check the prerequisites.
+Over-provisioning CPU, RAM and disk will give you more headroom.
+
+### What is the Blockfrost Platform `server-address` configuration parameter used for?
+
+The `server-address` configuration parameter defines the IP address and port that your Icebreaker node's API server listens on.
+
+By default this uses `0.0.0.0:3000` which listens on all IPv4 interfaces on port 3000.
+
+Common customizations:
+
+- To use IPv6, specify `::` as the IP
+- To limit to localhost only, use `127.0.0.1`
+- To use a different port, change the `:3000` portion
+
+Be cautious when changing this value as it can impact your node's public reachability for API requests.
+Always ensure the specified interface and port are publicly accessible.
+
+## Operation and monitoring
+
+### How can I monitor and verify that my Icebreaker node is working correctly?
+
+Please see the [Verifying that you are up and running](/installation/verification) guide.
+
+### How can I check if my Icebreaker node is active?
+
+To verify if your Icebreaker node is active, you can:
+
+1. Access the [Icebreaker dashboard](https://blockfrost.grafana.net/public-dashboards/8d618eda298d472a996ca3473ab36177) to check the status of your node.
+2. Use the command line to check logs for successful registration messages:
+   ```bash
+   blockfrost-platform --config /etc/cardano/mainnet/blockfrost.toml
+   ```
+   Look for "Successfully registered with Icebreakers API."
+
+### Where can I find the Icebreaker dashboard and monitor system metrics?
+
+The [Icebreaker dashboard](https://blockfrost.grafana.net/public-dashboards/8d618eda298d472a996ca3473ab36177) provides real-time information about the Icebreaker network, including:
+
+- Total number of minted Icebreaker NFTs
+- Number of Icebreaker nodes currently active
+- Percentage of active nodes compared to total slots
+- Current distribution of `blockfrost-platform` versions in use
+- Icebreakers by Status: Current count of nodes in 'Up' vs 'Down' status based on the last health check
+- Status check duration: Time in milliseconds for a node to respond to the last status check ping
+- Average total response time: Mean response time over the last 10 minutes for API requests to each node
+- Incoming/Outgoing bytes: Network traffic to and from each node, indicating overall activity level
+
+This dashboard serves as the central monitoring location for all Icebreaker-related metrics and is regularly updated as nodes join or leave the network.
+
+Use these metrics to monitor your node's availability, performance and traffic levels relative to other nodes.
+Tooltips explain each metric.
+
+### What endpoints are currently supported by the Blockfrost platform?
+
+Currently, only the 'submit' endpoint is implemented in the Blockfrost platform. Additional endpoints are planned for future releases.
```

Comment:
Is this in comparison to the Blockfrost.io hosted service?
If so, It would be nice to mention so.
If not we have the /metrics and the root / endpoints as well.

Reply:
@ginnun Would it be accurate to say this: 

Currently, the Blockfrost platform supports these endpoints: 

- submit
- metrics
- root

Additional endpoints are planned for future releases.